spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigMapPropertySourceTests.java

```diff
@@ -42,15 +39,13 @@ class Fabric8ConfigMapPropertySourceTests {
 
 	private KubernetesClient mockClient;
 
-	private final DefaultKubernetesClient client = Mockito.mock(DefaultKubernetesClient.class);
```

Comment:
we were relying on an implementation detail of the fabric8 extension, essentially before this upgrade, the extension would not touch `DefaultKubernetesClient`. Now they have made the implementation more strict, as such I have moved these tests to a different class. I do this in various other places too.